FlowCal/transform.py

```diff
@@ -118,6 +128,15 @@ def to_rfi(data, channels=None, max_range=None, amplification_type=None):
         the first element is zero, the amplification type is linear. If
         None, take `amplification_type` from
         ``data.amplification_type(channel)``.
+    amplifier_gain : float or list of floats, optional
+        The linear amplifier gain of the specified channel(s). Only used if
+        ``amplification_type[0]==0`` (linear amplifier). If None,
+        take `amplifier_gain` from ``data.amplifier_gain(channel)``. If
+        `data` does not contain ``amplifier_gain()``, use 1.0.
+    max_range : int, float, or list of int or float, optional
+        Maximum range, for each specified channel. Only needed if
+        ``amplification_type[0]!=0`` (log amplifier). If None, take
+        `max_range` from ``len(data.domain(channel))``.
```

Comment:
On second thought, it may be worth changing this to "resolution", since that what `r` in the equation is supposed to be. Also, on line 232 and 233, we are taking `max_range` from `data.resolution` if it is not available. Changing this to `resolution` would make it consistent with the rest of our nomenclature, and preserve the nice pattern that this function has in which the argument names correspond to the property names of `FCSData` that they're taken from when they're not specified.

tl;dr: `max_range` should be change to `resolution`.